components/Starknet/modules/architecture_and_concepts/pages/Smart_Contracts/serialization_of_types_in_Cairo.adoc

```diff
@@ -270,10 +270,13 @@ The calldata for this struct is serialized as follows: `0,2,5,3,1,2,3`
 A string is represented in Cairo as a `ByteArray` type. The first byte of each word in the byte array is the most significant byte in the word. A byte array has the following structure:
 
 [horizontal]
-. *1st member*: The number of 31-byte words in the array construct.
-. *middle members*: The data. One or more `felt252` values, each containing at most 31 bytes. where the last, or only, value is less than or equal to 30 bytes.
-+
+. **First Member: Word Count** 
```

Comment:
Let's make it:

. *Word count:* <explanation>
. *Data:* <explanation>
. The number of bytes of the pending word.